karma.py

```diff
@@ -47,11 +47,20 @@ def karma(self, event):
     @cmd
     def top_karma(self, event):
         """Get 5 people with most karma points."""
+        args = event.args
         output = ""
         inverted = [(value, key) for (key, value) in self.karmas.items()]
         karmees = sorted(inverted, reverse=True)
-        # Takes top 5 or less if len(karmees) < 5
-        karmees = karmees[:5]
+
+        n = 5
+        if len(args) == 1:
```

Comment:
@JakubN well, if I run this with more than one argument I'd still expect the first one to be tsken into account, rather than defaulting n to 5.